maro/cli/inspector/visualization_choice.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+class PanelViewChoice(Enum):
+    Intra_Epoch = 1
+    Inter_Epoch = 2
+
+
+class CIMIntraViewChoice(Enum):
+    by_port = 1
```

Comment:
why not add by_vessel?

Reply:
Because first of all, I think in cim senario, port is the more important resource holder. Secondly, the classification of by port/by snapshot is actually based on port data. If we join the choice by vessel, it means that we will add another level of selection, and I think that the vessel data can be displayed more adequately according to snapshot and sampling, so this item is not added.